Add versioning (#35)
jellyfish-dev/jellyfish-docs: README.md

The docs for the [Jellyfish](https://github.com/jellyfish-dev/jellyfish).
You can read the docs here: https://jellyfish-dev.github.io/jellyfish-docs/

## Installation

```
npm install
```

## Local Development

```
npm start
```

This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.

## Build

```
npm run build
```

This command generates static content into the `build` directory and can be served using any static contents hosting service.

## Versioning

> We follow the versioning system provided by docusaurus: [click](https://docusaurus.io/docs/versioning)

Reviewer: ```suggestion
We follow [the versioning system provided by docusaurus](https://docusaurus.io/docs/versioning)
```

To create a new docs version:

> * make sure the current docs version inside `/docs` directory and `sidebars.js` are ready to be frozen

Reviewer: I'd rephrase that, the way it is right now it made me look all over `/docs` and `sidebars.js` for a version string like `0.2.0` :smile:
```suggestion
* make sure that `sidebars.js` and files inside `docs/` directory are up-to-date and ready to be frozen
```
